### ISSUE TITLE
when pressing NVDA+f5, NVDA should update document and put in the same position

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2032]
**Reported by drein on 2012-01-06 18:35**
When we are in browse mode, and press nvda+f5, NVDA bring the virtual cursor to the top of document.
When possible, it should be suitable to have the cursor to the same position before pressing NVDA+f5.

**Comment by bhavyashah** [comment]
@jcsteh Your https://github.com/nvaccess/nvda/issues/2032#issuecomment-155295154 is slightly unclear to me. Pressing NVDA+F5 always dislaces the focus to the top of the document rather than there being particular cases where this bug takes place. Could you please clarify?

**Comment by jcsteh** [comment]
My point is that this command is more meant for diagnosing problems, rather than something that users should actually use in real world usage. So, having it return focus to the right place isn't something we consider important because it shouldn't be necessary in real world usage anyway. If there are cases where you need to use NVDA+f5, those should be filed specifically and addressed.

**Comment by bhavyashah** [comment]
@jcsteh Okay, but if specific cases should be filed separately, can we close this ticket?

**Comment by jcsteh** [comment]
That's a fair point. Closing as won't fix.
